src/components/ResultsList.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+class ResultsList extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			results: [],
+		};
+		
+		this.displayList = this.displayList.bind(this);
+		this.tableSort = this.tableSort.bind(this);
+	}
+
+	componentWillReceiveProps(nextProps) {
+		this.setState({ results: nextProps.results });
+	}
+
+	displayList() {
+		if (this.state.results.length === 0) {
+			return (
+				<tr>
+					<td colspan="5"><p>There are no results to display.</p></td>
+				</tr>
+			);
+		}
+
+		return this.state.results.map(result => {
+			return (
```

Comment:
think it would be worth moving this to it's own component, passing the data via props

Reply:
Rename in app.js resolved this.